package.json

```diff
@@ -36,7 +36,7 @@
 		"@nextcloud/auth": "^1.2.3",
 		"@nextcloud/axios": "^1.3.2",
 		"@nextcloud/capabilities": "^1.0.2",
-		"@nextcloud/dialogs": "^1.3.0",
+		"@nextcloud/dialogs": "^1.4.0",
```

Comment:
are you trying to sneak something in? seems quite unrelated to "fix tabs & lint", no?

Reply:
Shhhhhh just approve 
![](https://media.giphy.com/media/XItRQJP0wai7m/source.gif)